.circleci/config.yml

```diff
@@ -13,8 +13,8 @@ post_checkout: &post_checkout
         fi
         if [[ $BASE_COMMIT ]]; then
           CHANGED=$(git diff --name-only $BASE_COMMIT)
-          echo $CHANGED
-          echo 'export CHANGED="$CHANGED"' >> $BASH_ENV
```

Comment:
Does bash do something different with single quotes?

Reply:
Shells, Perl, and Ruby allow almost no escapes and no interpolation in single-quoted strings. (In contrast to e.g. Python and JavaScript which treat single and double-quotes similarly.)